strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java

```diff
@@ -59,6 +59,7 @@ public final void setObjectId(String objectId)
         this.objectId = objectId;
     }
 
+    @XmlTransient
```

Comment:
@fuss86 should it be transient?

Reply:
Yeah, haven't noticed it before. I think XML file shouldn't have any extra information from our database so `@XmlTransient` would is expected here. Thanks.

Reply:
ok
thanks for your response